aws/flatcar-linux/kubernetes/security.tf

```diff
@@ -42,36 +42,40 @@ resource "aws_security_group_rule" "controller-etcd-metrics" {
   source_security_group_id = "${aws_security_group.worker.id}"
 }
 
-resource "aws_security_group_rule" "controller-apiserver" {
-  security_group_id = "${aws_security_group.controller.id}"
+resource "aws_security_group_rule" "controller-vxlan" {
+  count = "${var.networking == "flannel" ? 1 : 0}"
 
-  type        = "ingress"
-  protocol    = "tcp"
-  from_port   = 6443
-  to_port     = 6443
-  cidr_blocks = ["0.0.0.0/0"]
-}
-
```

Comment:
Uneeded diff? Seems the same is added at the end

Reply:
Added from https://github.com/poseidon/typhoon/commit/af18296bc5ee8bf69745d236444df7a9d44c4050

Reply:
Ohh, okay. This is really useful to review